src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -47,6 +57,24 @@ const FilterConditionSelector: React.FC<Props & WithOutsideClickProps> = ({
 	const listRef = React.createRef<HTMLUListElement>();
 	const mainRef = React.useRef<HTMLElement | null>(null);
 
+	React.useEffect(() => {
+		return () => {
+			clearAllBodyScrollLocks();
+		};
+	});
```

Comment:
На каждый апдейт нужно клирить? Если хотел на анмаунт, то забыл зависимость проставить
```suggestion
	}, []);
```

Reply:
да, нужно на каждый апдейт